.github/workflows/maven.yml

```diff
@@ -15,11 +16,15 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up JDK 11
-      uses: actions/setup-java@v2
-      with:
-        java-version: '11'
-        distribution: 'adopt'
-    - name: Build with Maven
-      run: mvn test
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - name: Install Google Chrome # Using shell script to install Google Chrome
+        run: |
+          chmod +x ./InstallChrome.sh
+           ./InstallChrome.sh
+      - name: Build with Maven
+        run: mvn test
```

Comment:
- [ ] guess linux likes a new line at end. that is what this red warning is telling us. Can you press enter (thus add a newline) and push change again?